README.md

```diff
@@ -1,4 +1,7 @@
-# NLP Best Practices
+<img src="NLP-Logo.png" align="right" alt="" width="300"/>
```

Comment:
@saidbleik is this a Microsoft approved logo? It seems like the same one from the public repo right. It looks great

Reply:
@sharatsc created this one for the github website